### PR TITLE
KOGITO:3788 - Jobs management sorting controls

### PR DESCRIPTION
**Pull request by Sara4994**
Hello,

Please review this PR, intended to add sorting controls on jobs management table.

Table data can be sorted by ascending or descending on all the columns except "ID". By default table is sorted in the ascending order of "Last update".

JIRA issue: https://issues.redhat.com/browse/KOGITO-3788

### CODE REVIEW

**Comment by Sara4994** [comment]
jenkins retest this please

**Review by AjayJagan — APPROVED** [review]
LGTM

**Comment by Sara4994** [comment]
jenkins retest this please

File: ui-packages/packages/common/index.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export { default as LoadMore } from './src/components/Atoms/LoadMore/LoadMore';
 export { default as DomainExplorer } from './src/components/Organisms/DomainExplorer/DomainExplorer';
 export { default as DomainExplorerListDomains } from './src/components/Organisms/DomainExplorerListDomains/DomainExplorerListDomains';
 export * from './src/utils/OuiaUtils';
+export * from './src/utils/Utils';
```
Comment (jstastny-cz) [inline ui-packages/packages/common/index.ts:20]: maybe just a small nit-pick ... we're exporting everything and then using function `set` in import. That's not a very descriptive name. Can we either rename it here in export, or at least use import {set as better-descriptive-name} from '@kogito-apps/common' when importing?

Comment (Sara4994) [reply ui-packages/packages/common/index.ts:20]: @jstastny-cz  done

**Review by jstastny-cz — APPROVED** [review]
LGTM, thanks.

**Review by nmirasch — APPROVED** [review]

**Pull request merged**